### ISSUE TITLE
bv_maker build issue

### ISSUE THREAD

**Issue by rbregand** [brainvisa/axon#54]
Hi, 
An issue occurs when I'm running "bv_maker build". You will find the log bellow : 

"""
-- Configuring incomplete, errors occurred!
See also "/casa/host/build/CMakeFiles/CMakeOutput.log".
See also "/casa/host/build/CMakeFiles/CMakeError.log".
Makefile:30556: recipe for target 'cmake_check_build_system' failed
make: *** [cmake_check_build_system] Error 1
Traceback (most recent call last):
  File "/casa/host/build/python/brainvisa/maker/commands.py", line 72, in process
    **meth_kwargs)
  File "/casa/host/build/python/brainvisa/maker/build.py", line 666, in build
    timeout=timeout)
  File "/casa/host/build/python/brainvisa/maker/subprocess.py", line 59, in system
    six.reraise(*error)
  File "/usr/local/lib/python3.6/dist-packages/six.py", line 718, in reraise
    raise value.with_traceback(tb)
  File "/casa/host/build/python/brainvisa/maker/subprocess.py", line 48, in system
    raise OSError()
OSError: Command failed: 'make' '-j8'
From directory: /casa/host/build
"""
This issue is not at his right place, but I don't know where to create it. Can you put it in the good section ?

Thanks in advance for your help.

**Comment by denisri** [comment]
The actual error is earlier, in the lines you didn't copy in your ticket... Moreover you seem to have done `bv_maker build`; has `bv_maker configure` been done first ? (the error seems in the configure step)

**Comment by rbregand** [comment]
You're right, I first ran bv_maker sources. I didn't pay attention but there was an error : 
"""
Updating git repository /casa/host/src/morphologist-gpl

ERROR: directory "/casa/host/src/morphologist-gpl" is not empty, Git will not be able to clone into it.
This error may be due to a repository change for a component (typically
going from Subversion to Git). You must check yourself that you have nothing
to keep in this directory and delete it to make "bv_maker sources" work.
$ git clone --branch master https://github.com/brainvisa/morphologist-gpl.git /casa/host/src/morphologist-gpl
fatal: destination path '/casa/host/src/morphologist-gpl' already exists and is not an empty directory.
Traceback (most recent call last):
  File "/casa/host/build/python/brainvisa/maker/commands.py", line 72, in process
    **meth_kwargs)
  File "/casa/host/build/python/brainvisa/maker/sources.py", line 283, in process
    gr.update_or_clone(source_dir=self)
  File "/casa/host/build/python/brainvisa/maker/git.py", line 473, in update_or_clone
    self.remote_url, self.path)
  File "/casa/host/build/python/brainvisa/maker/subprocess.py", line 59, in system
    six.reraise(*error)
  File "/usr/local/lib/python3.6/dist-packages/six.py", line 718, in reraise
    raise value.with_traceback(tb)
  File "/casa/host/build/python/brainvisa/maker/subprocess.py", line 48, in system
    raise OSError()
OSError: Command failed: 'git' 'clone' '--branch' 'master' 'https://github.com/brainvisa/morphologist-gpl.git' '/casa/host/src/morphologist-gpl'

bv_maker summary:
FAILED      step sources: /casa/host/src, started: 2021/11/08 13:12, stopped: 2021/11/08 13:12
"""

I just just did as it says : remove the folder. After that everything went all good for configure and built steps.
I will close the issue.
